test/tsconfig.json

```diff
@@ -6,7 +6,8 @@
       "mocha"
     ],
     "lib": [
-      "esnext"
+      "esnext",
+      "dom"
```

Comment:
Why add this back? These tests don't run in the browser and so don't have access to DOM apis.

Reply:
used in [https://github.com/elastic/kibana/pull/35355/files#diff-89e782accedd07fd38aed70772972711R407](400 line)

Reply:
ugh, I really wish TS was less reliant on global types... fine